pkgs/tools/virtualization/nixos-container/default.nix

```diff
@@ -1,4 +1,10 @@
-{ substituteAll, perlPackages, shadow, utillinux }:
+{ substituteAll
+, perlPackages
+, shadow
+, utillinux
+, configurationDirectory ? "/etc/containers"
+, stateDirectory ? "/var/lib/containers"
```

Comment:
I think that we should use the "new" default here, i.e. `/var/lib/nixos-containers` (and `/etc/nixos-containers`).